examples/publisher/pub_delay/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"log"
+	"time"
+
+	"beanq"
+	"beanq/helper/json"
+	opt "beanq/internal/options"
+	"github.com/spf13/cast"
+)
+
+func main() {
+	pubDelayInfo()
+}
+
+func pubDelayInfo() {
+	pub := beanq.NewPublisher()
+
+	m := make(map[string]any)
+
+	for i := 0; i < 10; i++ {
+		y := 0
+		m["delayMsg"] = "new msg" + cast.ToString(i)
+		m["a"] = "sfdsf"
+		m["b"] = "bbbb"
+		m["c"] = "ccccc"
+		m["d"] = "sdfsfdsfsf"
+		m["e"] = "sdfsfsfsf"
+
+		b, _ := json.Marshal(m)
+
+		task := beanq.NewTask(b, beanq.SetName("update"))
+		delayT := time.Now().Add(100 * time.Second)
+		if i == 2 {
+			delayT = time.Now()
+		}
+		if i == 3 {
+			y = 10
+			delayT = time.Now().Add(35 * time.Minute)
+		}
+
+		// This part is for the convenience of future testing, so keep it for now
+		/*
+			if i > 20 {
+				delayT = time.Now().Add(25 * time.Second)
+			}*/
```

Comment:
// TODO: This part is for the convenience of future testing, so keep it for now
// if i > 20 {
//    delayT = time.Now().Add(25 * time.Second)
//}

Reply:
have deleted it